packages/mui-material-pigment-css/package.json

```diff
@@ -43,7 +43,7 @@
     "@mui/system": "workspace:*"
   },
   "peerDependencies": {
-    "@pigment-css/react": "^0.0.27 || ^0.0.28"
+    "@pigment-css/react": "^0.0.29"
```

Comment:
I'm not sure why this was set with an `||` operator in https://github.com/mui/material-ui/pull/44611. Is this change correct @brijeshb42?

Reply:
Thanks! May I ask you to review and/or approve this PR? 😊